features/netsocket/emac-drivers/TARGET_WHD/interface/WhdSTAInterface.h

```diff
@@ -0,0 +1,204 @@
+/* WHD implementation of NetworkInterfaceAPI
+ * Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef WHD_STA_INTERFACE_H
+#define WHD_STA_INTERFACE_H
+
+#include "mbed.h"
```

Comment:
include what is needed, not `mbed.h` file

Reply:
Replaced with:
```
#include "netsocket/WiFiInterface.h"
#include "netsocket/EMACInterface.h"
```